Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented:
@GioviQ - I see you updated the Italian Account UI theme, and added double single quotes, and added an automatic check in the build (unfortunately I can't add such a check in Weblate). 

There's been a recent change to the guidelines that this should only be done in the base themes, see https://github.com/keycloak/keycloak/blob/main/docs/translation.md#handling-of-single-quotes-

I've reverted the changes about the double quotes.

weblate committed:
Updated translation for Japanese

Language: ja

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Kohei Tamura <ktamura.biz.80@gmail.com>

weblate committed:
Updated translation for Romanian

Language: ro

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: niwde <tuzaredwin@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: niwde <tuzaredwin@gmail.com>

weblate committed:
Updated translation for Spanish

Language: es

Co-authored-by: Hernan Lopez <hernan.lopez@gmail.com>
Signed-off-by: Hernan Lopez <hernan.lopez@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Updated translation for German

Language: de

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

weblate committed:
Update translation files

Updated by "Remove blank strings" hook in Weblate.

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Updated translation for Italian

Language: it

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Giovanni Quarella <gquarella@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Giovanni Quarella <gquarella@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Kohei Tamura <ktamura.biz.80@gmail.com>

ahus1 commented:
@liviuroman / @edwint88: The Romanian changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@robson90: The German changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@y-tabata / @wadahiro: The Japanese changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@herver1971 / @anthieni: The Spanish changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@EdoardoTona / @GioviQ: The Italian changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.
